pkg/operator/ceph/cluster/rbd/mirror.go

```diff
@@ -133,33 +132,43 @@ func (r *ReconcileCephRBDMirror) removeExtraMirrors(cephRBDMirror *cephv1.CephRB
 		return errors.Wrap(err, "failed to get mirrors")
 	}
 
```

Comment:
How about a comment that this method to remove extra mirrors will no longer be needed after v1.6? Then we can more easily remove the dead code in the future.

Reply:
I've opened https://github.com/rook/rook/issues/7183 instead for better tracking!